.pylintrc

```diff
@@ -229,7 +229,7 @@ max-nested-blocks=5
 
 [FORMAT]
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=120
```

Comment:
Curious why we are changing these rules?

Reply:
is there an alternative to split the line? because otherwise you might need to update tox configuration around black and maybe even documentation.

https://github.com/aws/sagemaker-python-sdk/blob/master/tox.ini#L133-L141

Reply:
Okay let me remove my change from here.